changelog.d/798.removal

```diff
@@ -0,0 +1 @@
+Drop support for the Sled crypto store format. The configuration key of `experimentalEncryption.useLegacySledStore` is now ignored, and the crypto store will always use the SQLite format. If an existing SQLite store does not exist on bridge startup, one will be created.
```

Comment:
Sorry, another opinion. Can we have the app fail to start if this key is present to force users to take action?

Reply:
Good idea